planning/standards-status.md

```diff
@@ -33,22 +34,22 @@ description: This list collects standards that the Thunderbird family of applica
+| List Management Headers | [rfc2369](https://datatracker.ietf.org/doc/html/rfc2369), [rfc5064](https://datatracker.ietf.org/doc/html/rfc5064) |        ✔       |        ✔       |                       |         |
 
 ### Security and Authentication
 
 | Spec Name       |                                                                                   Spec URLs                                                                                  | Desktop Status | Android Status | Standards Involvement        | Comment                                                     |
 | --------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------: | :------------: | ---------------------------- | ----------------------------------------------------------- |
 | MIME            | [rfc2045](https://datatracker.ietf.org/doc/html/rfc2045), [rfc2046](https://datatracker.ietf.org/doc/html/rfc2046), [rfc2047](https://datatracker.ietf.org/doc/html/rfc2047) |        ✔       |        ✔       |                              | All 3 are supported for both Desktop and Android platforms. |
-| OpenPGP         | [rfc4880](https://datatracker.ietf.org/doc/html/rfc4880)                                                                                                                                                                             |        ✔       |        ✔       | Kai Engert actively involved |                                                             |
-| S/MIME version? |                                                                                                                                                                              |        ✔       |                |                              |                                                             |
-| oAuth 2.0       |                                                           [rfc6749](https://datatracker.ietf.org/doc/html/rfc6749)                                                           |        ✔       |        ✔       |                              | See also [https://oauth.net/2/](https://oauth.net/2/)                               |
+| OpenPGP         | [rfc4880](https://datatracker.ietf.org/doc/html/rfc4880)                                                                                                                     |        ✔       |        ✔       | Kai Engert actively involved |                                                             |
+| S/MIME version? |                                                                                                                                                                              |        ✔       |        ✖       |                              |                                                             |
```

Comment:
Does the "x" indicate that it's not supported *yet*?

Reply:
Yes. I added an entry to the legend. See https://github.com/thunderbird/developer-docs/pull/193/files#diff-d08118a0a32b41be3fc6fca82f7445e923208c4245b26168a2d9b1b337981993R10

Reply:
oh you sure did and I missed it when I looked at the changes! Thanks so much :)